### PR TITLE
feat: theming dropdown

### PR DESCRIPTION
**Pull request by zopa-charlotte-walker**

### CODE REVIEW

File: src/components/molecules/DropdownFiltered/DropdownFiltered.tsx [hunk 1]
```diff
@@ -44,10 +44,9 @@ export const SearchInputWrap = styled.div`
 
 export const CustomIcon = styled.div<{ isOpen: boolean }>`
   background: transparent url(${unbrandedChevron}) no-repeat center;
-  background-size: 60%;
+  background-size: 120%;
```
Comment (zopa-charlotte-walker) [inline src/components/molecules/DropdownFiltered/DropdownFiltered.tsx:47]: don't think this change has affected anything anywhere else but cant be sure - struggled passing theming into this component for some reason

File: src/components/molecules/DropdownFiltered/DropdownFiltered.tsx [hunk 2]
```diff
@@ -123,6 +122,7 @@ const DropdownFiltered = (props: DropdownFilteredProps) => {
                   endIcon={
                     theme.input.searchInput.customIcon ? (
                       <CustomIcon
+                        color={theme.input.searchInput.customIconColor}
```
Comment (zopa-charlotte-walker) [inline src/components/molecules/DropdownFiltered/DropdownFiltered.tsx:125]: not sure this works passing through like this :thinking:

File: src/components/molecules/DropdownFiltered/DropdownFiltered.tsx [hunk 3]
```diff
@@ -123,6 +122,7 @@ const DropdownFiltered = (props: DropdownFilteredProps) => {
                   endIcon={
                     theme.input.searchInput.customIcon ? (
                       <CustomIcon
+                        style={{ color: theme.input.searchInput.customIconColor }}
```
Comment (zopa-charlotte-walker) [inline src/components/molecules/DropdownFiltered/DropdownFiltered.tsx:125]: can either do this or just set it to the colour hard coded in the CustomIcon component

File: src/components/atoms/InputText/InputText.tsx [hunk 4]
```diff
@@ -44,7 +44,7 @@ const Input = styled.input<InputThemeProps>`
   padding: ${({ theme }: InputThemeProps) => theme.input.padding};
   padding-left: ${({ startIcon, theme }: InputThemeProps) =>
     startIcon && theme.input.startIcon ? theme.input.startIconPaddingLeft : null};
-  padding-right: ${({ endIcon }: InputThemeProps) => !!endIcon && '60px'};
+  padding-right: ${({ endIcon, theme }: InputThemeProps) => endIcon && theme.input.startIconPaddingRight};
```
Comment (MichelleXBai) [inline src/components/atoms/InputText/InputText.tsx:47]: Double checking:  does it mean to be `startIconPaddingRight` as it passed in endIcon

Comment (zopa-charlotte-walker) [reply src/components/atoms/InputText/InputText.tsx:47]: good spot! youre right

**Review by admmasters — APPROVED** [review]

**Pull request merged**